udata/templates/dataset/list.html

```diff
@@ -43,7 +47,15 @@
 
 {% block main_content %}
 <div class="row">
-    <div class="col-md-8 col-lg-9 smaller">
+
+		{% if not datasets %}
```

Comment:
Express the opposite, it should be clearer: `if datasets` ... `else` ... (positive condition and nominal case first)